srcpkgs/jcal/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'jcal'
+pkgname=jcal
+version=0.4.1
+revision=1
+build_wrksrc="sources"
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+makedepends="readline-devel"
+short_desc="Jalali Calendar Library"
+maintainer="linarcx <linarcx@gmail.com>"
+license="GPL-3.0"
```

Comment:
this should be `GPL-3.0-or-later`

Reply:
How you find that?

Reply:
I looked at the source files, e.g. jcal.c states:

```
 * jcal is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
```

`GPL-3.0-or-later` is the correct SPDX identifier for that case.

Reply:
Thank you for clarification.
Done!